types/ws/index.d.ts

```diff
@@ -82,7 +82,7 @@ declare namespace WebSocket {
 
     type VerifyClientCallbackSync = (info: { origin: string; secure: boolean; req: http.IncomingMessage }) => boolean;
     type VerifyClientCallbackAsync = (info: { origin: string; secure: boolean; req: http.IncomingMessage }
-        , callback: (res: boolean) => void) => void;
+        , callback: (res: boolean, code?: number, message?: string) => void) => void;
```

Comment:
Hi @mhegazy , thanks for suggestion!

But if I set the rest parameters required, declaration is not backward compatible. All the existing codes cannot pass the compilation.